webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -218,31 +222,42 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
     );
   }
 
-  private getUserHomePlaceSummary(homePlaceId?: string) {
+  private getUserFacilityId(userSettings) {
+    return Array.isArray(userSettings.facility_id) ? userSettings.facility_id : [userSettings.facility_id];
+  }
+
+  private getUserHomePlaceSummary() {
     return this.userSettingsService
       .get()
-      .then((userSettings:any) => {
-        const facilityId: string = homePlaceId ?? userSettings.facility_id;
-        if (!facilityId) {
+      .then((userSettings: any) => {
+        const facilityId = this.getUserFacilityId(userSettings);
+
+        if (!facilityId.length || facilityId.some(id => id === undefined)) {
```

Comment:
I don't understand. We're not loading the home place summary if any of the places are undefined? 
I have trouble understanding the logic here.

Reply:
This was change was I related to the base branch, not this PR. 

When working with strings the check was `if (!facilityId)`, when we changed to Arrays, I changed it to `if(!facilityId.length)` but I realized there are situations where we have an array with `[undefined]` and that check fails. That's why I added that if the element in the Array is `undefined` then disregard it.